data/org.gnome.GTG.desktop.in

```diff
@@ -9,6 +9,7 @@ Exec=gtg %U
 Terminal=false
 Type=Application
 Categories=Office;ProjectManagement;
-Keywords=Organizer;Task;Activity;Plan;Time;GTD;gtg;Todo;
+# Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+Keywords=Organizer;Task;Tasks;Projects;Activities;Productivity;Plan;Planning;Planner;Time;Management;Scheduling;GTD;gtg;Todo;
```

Comment:
This feels to be going a bit overboard with the keywords.
As far as gnome-shell is concerned, for eaxmple "Task" is unnecessary when you have "Tasks", as the search would still find "Task" from "Tasks". But I guess translations might change this, as languages may be changing the ending when doing plural, etc.
But in general I think this should be toned down a bit to what we have as keywords, which was more of a problem in the original version too already, but more evident now that some different forms and plurals are added.

Reply:
I thought so as well, and then thought "right, but it might still be useful to keep the base words around because not every language constructs itself like English" as you pointed out... I'm guessing there is no bad consequences to having more keywords than not, in this case.